docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -62,9 +66,9 @@ Finally start your plugin in development mode. Go to your plugin root directory
    yarn watch
    ```
 
-2. Start Grafana backend and frontend:
+2. Start the Grafana backend and frontend:
```

Comment:
are these steps still accurate? They conflict with those in the developer guide...

Reply:
Right, I'm not sure. I'll ask a developer to (re-)review this section.

Reply:
Esteban said that the instructiosn aren't inaccurate because there are different ways of getting set up. The trouble is that the docs aren't opinionated enough, and we need to specify one way (best way) to get it done. Tracking in https://github.com/grafana/grafana/issues/68500